RubberduckTests/QuickFixes/AccessSheetUsingCodeNameQuickFixTests.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System.Linq;
+using System.Threading;
+using Moq;
+using NUnit.Framework;
+using Rubberduck.Inspections.Concrete;
+using Rubberduck.Inspections.QuickFixes;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.VBEditor.SafeComWrappers;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using RubberduckTests.Mocks;
+
+namespace RubberduckTests.QuickFixes
+{
+    [TestFixture]
+    public class AccessSheetUsingCodeNameQuickFixTests
```

Comment:
Same comments RE: testing for implicit `WorkSheets` reference and `Application.WorkSheets`.